src/containers/add-course-banner/AddCourseBanner.styles.ts

```diff
@@ -0,0 +1,27 @@
+export const getStyles = (bannerUrl: string) => ({
```

Comment:
rename getStyles to styles and pass bannerUrl not to the whole styles object, but only to the container property

Reply:
Done